core/src/main/java/org/infinispan/remoting/transport/jgroups/JGroupsTransport.java

```diff
@@ -478,7 +478,7 @@ private void setSiteMasterPicker(SiteMasterPickerImpl siteMasterPicker) {
       }
    }
 
-   private void startJGroupsChannelIfNeeded() {
+   protected void startJGroupsChannelIfNeeded() {
```

Comment:
Please also add a javadoc so we don't remove it again in the future.

Reply:
@danberindei done.